scripts/open-coredump.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+# Script for "one-click" opening of coredumps.
+# It extracts the build-id from the coredump, retrieves metadata for that
+# build, downloads the binary package, the source code and finally
+# launches the dbuild container, with everything ready to load the coredump.
+# The script is reentrant: running it after the prepartory steps will re-use what
+# is already donwloaded. It is *strongly* recommended to run this from an empty
+# directory, with nothing but the core-file present.
+#
+# Requirements:
+# * eu-unstrip
+# * curl
+# * jq
+# * git
+
+set -e
+
+SCYLLA_S3_RELOC_SERVER_URL="http://backtrace.scylladb.com"
+
+if [[ $# != 1 ]]; then
+	echo Please provide the path to the coredump
+	exit 1
+fi
+
+for required in eu-unstrip jq curl git; do
+	if ! type $required >& /dev/null; then
+		echo Please install $required first
+		exit 1
+	fi
+done
+
+COREFILE=$1
+
+BUILD_ID=$(eu-unstrip -n --core ${COREFILE} | cut -f2 -d' ' | cut -f1 -d@)
```

Comment:
`cut` should also be checked in the `required` list above.

Reply:
Isn't that a standard POSIX tool? Should be present in all systems.

Reply:
It's part of the `coreutils` rpm in redhat distros so it's typically installed by default along with cat, cp, ls, mv, and friends.